app/src/main/java/org/mozilla/fenix/home/HomeFragmentStore.kt

```diff
@@ -264,3 +274,13 @@ internal fun List<RecentlyVisitedItem>.filterOut(groupTitle: String?): List<Rece
         false -> this
     }
 }
+
+/**
+ * Removes a [RecentTab.Tab] from a list of [RecentTab]. [RecentTab.SearchGroup]s will not be filtered.
+ *
+ * @param tab [RecentTab] to remove from the list
+ */
+@VisibleForTesting
+internal fun List<RecentTab>.filterOutTab(tab: RecentTab): List<RecentTab> = filterNot {
```

Comment:
Can we move this into /ext/RecentTab

Reply:
Sure! Should I move the similar extension above it into /ext/RecentlyVisited while I'm at it?